int8 person_id
time latest_time
geometry_msgs/TransformStamped[] body_parts
